src/explore-education-statistics-admin/src/pages/release/data/ReleaseApiDataSetChangelogPage.tsx

```diff
@@ -74,49 +78,55 @@ export default function ReleaseApiDataSetChangelogPage() {
       >
         Back to API data set details
       </Link>
+
       <LoadingSpinner loading={isLoadingDataSet || isLoadingChanges}>
-        <div className="govuk-grid-row">
-          <div className="govuk-grid-column-three-quarters">
-            <span className="govuk-caption-l">API data set changelog</span>
-            <h2>{dataSet?.title}</h2>
-          </div>
-        </div>
+        {!errorFetchingDataSet && !errorFetchingChanges ? (
+          <>
+            <div className="govuk-grid-row">
+              <div className="govuk-grid-column-three-quarters">
+                <span className="govuk-caption-l">API data set changelog</span>
+                <h2>{dataSet?.title}</h2>
+              </div>
+            </div>
+            <TagGroup className="govuk-!-margin-bottom-7">
+              <Tag colour={isDraft ? 'green' : 'blue'}>{`${
+                isDraft ? 'Draft' : 'Published'
+              } v${dataSetVersion?.version}`}</Tag>
+              <Tag
+                colour={dataSetVersion?.type === 'Major' ? 'blue' : 'grey'}
+              >{`${dataSetVersion?.type} update`}</Tag>
+            </TagGroup>
 
-        <TagGroup className="govuk-!-margin-bottom-7">
-          <Tag colour={isDraft ? 'green' : 'blue'}>{`${
-            isDraft ? 'Draft' : 'Published'
-          } v${dataSetVersion?.version}`}</Tag>
-          <Tag
-            colour={dataSetVersion?.type === 'Major' ? 'blue' : 'grey'}
-          >{`${dataSetVersion?.type} update`}</Tag>
-        </TagGroup>
+            {isDraft && showForm ? (
+              <ApiDataSetGuidanceNotesForm
+                notes={dataSetVersion?.notes}
+                onSubmit={handleUpdateNotes}
+              />
+            ) : (
+              <>
+                <h3>Public guidance notes</h3>
+                <p>
+                  {dataSetVersion?.notes ||
+                    'No notes have been added for this API data set.'}
+                </p>
+                {isDraft && (
+                  <Button onClick={toggleShowForm.on}>
+                    Edit public guidance notes
+                  </Button>
+                )}
+              </>
+            )}
 
-        {isDraft && showForm ? (
-          <ApiDataSetGuidanceNotesForm
-            notes={dataSetVersion?.notes}
-            onSubmit={handleUpdateNotes}
-          />
-        ) : (
-          <>
-            <h3>Public guidance notes</h3>
-            <p>
-              {dataSetVersion?.notes ||
-                'No notes have been added for this API data set.'}
-            </p>
-            {isDraft && (
-              <Button onClick={toggleShowForm.on}>
-                Edit public guidance notes
-              </Button>
+            {changes && dataSetVersion && (
+              <ApiDataSetChangelog
+                majorChanges={changes.majorChanges}
+                minorChanges={changes.minorChanges}
+                version={dataSetVersion.version}
+              />
             )}
```

Comment:
Would move the 'Could not load changelog' error message to be part of this conditional

Reply:
Done ta